www/xyerv00/DU2_firma.html

```diff
@@ -22,16 +22,16 @@ <h1>Light for  s.r.o. - více než světlo do vašeho domova</h1>
         <p>Společnost Light for home se zabývá dovozem lustrů, stojacích a nástěnných lamp a světidel od evropského výrobce. Všechny výrobky jsou vyráběny z těch nejkvalitnějších materiálů ruční výrobou. Skleněná stínidla jsou vyráběná skláři vyškolenými v Itlii. V nabídce je široké spektrum designů a stylů. Každý zákazník si u nás přijde na své.</p>
     </article>
     <h3>Naše nabídka</h3>
-    <dl>
-        <dt>Lustry</dt>
-        <dd> - závěsné i přímo přisazené ke stropu</dd>
-        <dt>Nástěnná svítidla</dt>
-        <dd> - všude, kde chcete mít osvětlenou plochu</dd>
-        <dt>Stojací lampy</dt>
-        <dd> - skvělé jako doplněk do interiéru</dd>
-        <dt>Stolní lampy</dt>
-        <dd> - pro práci i třeba na noční stolek...</dd>
-    </dl>
+    <ul>
+        <li>Lustry</li>
+        <p>závěsné i přímo přisazené ke stropu</p>
```

Comment:
přímo pod `<ul>` lze mít pouze `<li>`, takhle je to nevalidní HTML, teprve pod `<li>` mohou být další tagy